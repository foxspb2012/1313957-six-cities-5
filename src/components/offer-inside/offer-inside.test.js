import React from 'react';
import OfferInside from './offer-inside';
import renderer from 'react-test-renderer';

const FEATURES = [`Dishwasher`, `Baby seat`];

describe(`Should OfferInside render correctly`, () => {
  it(`Should OfferInside render correctly with features`, () => {
    const tree = renderer
      .create(
          <OfferInside
            features={FEATURES}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Should OfferInside render correctly without features`, () => {
    const tree = renderer
      .create(
          <OfferInside />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
