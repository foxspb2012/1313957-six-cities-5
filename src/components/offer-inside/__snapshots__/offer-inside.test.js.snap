// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should OfferInside render correctly Should OfferInside render correctly with features 1`] = `
<div
  className="property__inside"
>
  <h2
    className="property__inside-title"
  >
    What's inside
  </h2>
  <ul
    className="property__inside-list"
  >
    <li
      className="property__inside-item"
    >
      Dishwasher
    </li>
    <li
      className="property__inside-item"
    >
      Baby seat
    </li>
  </ul>
</div>
`;

exports[`Should OfferInside render correctly Should OfferInside render correctly without features 1`] = `
<div
  className="property__inside"
>
  <h2
    className="property__inside-title"
  >
    What's inside
  </h2>
  <ul
    className="property__inside-list"
  />
</div>
`;
